src/recommendations/RecommendationsList.jsx

```diff
@@ -15,6 +16,17 @@ const RecommendationsList = (props) => {
     title, recommendations, userId, setSelectedRecommendationsType, selectedRecommendationsType,
   } = props;
 
+  useEffect(() => {
```

Comment:
Why is this hook not in the `recommendationsPage` component? According to our approach, all setup-related things are in the `recommendationsPage` component, and the `recommendationsList` component's responsibility is to simply render and display recommendations.